docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,120 @@
+[[alert-schema]]
+
+== Alert schema
+
+{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices.  In 7.x versions, the index pattern was `.siem-signals-<spacename>`.  In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>`, and the field names have changed. The following table includes the new names and provides a cross-reference to the previous field names.
+
+NOTE: Users are advised NOT to use the `_source field` in alert documents, but rather use the fields API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].
+The non-ECS fields listed below are [beta] and subject to change.
```

Comment:
Should this line be a separate paragraph? It renders as part of the previous paragraph, which kind of loses impact. Convert to a [note block](https://github.com/elastic/docs#admon-blocks)?

Also, should it be `beta::[]` to render with the beta admonition?